### PR TITLE
fix: add aria-label for DefinitionList help popover

### PR DESCRIPTION
**Pull request by Kyzyl-ool**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -123,4 +123,4 @@
       "prettier --write"
     ]
   }
-}
+}
```
Comment (amje) [inline package.json:126]: Your editor is removing last new line, please disable this)

File: src/components/DefinitionList/components/Term.tsx [hunk 2]
```diff
@@ -19,12 +20,25 @@ function NoteElement({note}: NoteElementsProps) {
...
         );
     }
 
     if (typeof note === 'object') {
-        return <HelpPopover className={popoverClassName} placement={['bottom', 'top']} {...note} />;
+        return (
+            <HelpPopover
+                className={popoverClassName}
+                placement={['bottom', 'top']}
+                {...note}
```
Comment (Raubzeug) [inline src/components/DefinitionList/components/Term.tsx:35]: let's do like
```
const {buttonProps, ...rest} = note
...
```

**Review by Raubzeug — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Kyzyl-ool) [reply package.json]: fixed my vscode settings)

File: src/components/DefinitionList/components/Term.tsx (hunk 2 above)
Comment (Kyzyl-ool) [reply src/components/DefinitionList/components/Term.tsx]: done

**Review by Raubzeug — APPROVED** [review]

**Review by amje — APPROVED** [review]

**Pull request merged**